libs/shared/console-shared/src/lib/application-settings/ui/application-settings-resources/application-settings-resources.tsx

```diff
@@ -21,25 +20,30 @@ import {
 
 export interface ApplicationSettingsResourcesProps {
   displayWarningCpu: boolean
-  application?: ApplicationEntity
+  service?: Exclude<AnyService, Helm | Database>
```

Comment:
```suggestion
  service: Helm | Database
```
and you can remove all `service` null checks

Reply:
ok seens it's Exclude my bad